layers/generate_layers.py

```diff
@@ -659,21 +590,29 @@ def roadHasBusLane(tags):
             if value and "designated" in value:
                 return True
 
+    # Handle https://wiki.openstreetmap.org/wiki/Tag:access=no
+    if tags.get("access") == "no":
+        for key in ["bus", "psv"]:
+            value = tags.get(key)
+            if value == "yes" or value == "designated":
+                return True
+
     # We're not handling highway=busway or other cases for service roads
     # designed exclusively for buses, because they're not intended for cyclists
     # or any other users.
 
     return False
 
 
-def generateGeojsonFromOSMFile(osmFilePath, outputFilepath):
+def convertPbfToGeoJson(pbfPath, geojsonPath, geometryType):
```

Comment:
We always want to be explicit about exporting points, linestrings, or polygons, so add it as a mandatory param